olm/olm.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+BASE_DIR=$(cd "$(dirname "$0")" && pwd)
```

Comment:
Add license and a description what this script does.

Reply:
Thanks. Done.